tests/unit/Deserializers/StatementDeserializerTest.php

```diff
@@ -103,7 +103,7 @@ public function deserializationProvider() {
 					'snaktype' => 'novalue',
 					'property' => 'P42'
 				),
-				'type' => 'claim'
+				'type' => 'statement'
```

Comment:
This should stay 'claim'  as we want to make sure that the old serialization still works. Otherwise it would be the same test case as below.